contracts/GolemNetworkTokenWrapped.sol

```diff
@@ -119,6 +119,9 @@ contract GolemNetworkTokenWrapped is Token {
     string public constant symbol = "GNTW";
     uint8 public constant decimals = 18;     // same as GNT
 
+    event BatchTransfer(address indexed from, address indexed to, uint256 value,
+        uint64 closureTime);
```

Comment:
closureTime values are not checked in tests

Reply:
Done